backend/app/src/main/java/ch/zhaw/studyflow/domain/calendar/SQLCalendarDao.java

```diff
@@ -0,0 +1,90 @@
+package ch.zhaw.studyflow.domain.calendar;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * SQL implementation of the CalendarDao interface.
+ * This class provides methods to perform CRUD operations on Calendar objects
+ * stored in a SQL database.
+ */
+public class SQLCalendarDao implements CalendarDao {
+    private final Connection connection;
+
+    public SQLCalendarDao(Connection connection) {
+        this.connection = connection;
+    }
+
+    @Override
+    public Calendar create(Calendar calendar) {
+        String sql = "INSERT INTO calendars (id, name, owner_id) VALUES (?, ?, ?)";
+        try (PreparedStatement stmt = connection.prepareStatement(sql)) {
+            stmt.setLong(1, calendar.getId());
```

Comment:
The user shouldn't set an ID when creating a calendar. The ID is given to the calendar by the database (auto-increment).